source/AsyncRgbLedAnalyzerSettings.h

```diff
@@ -10,18 +10,33 @@ class AsyncRgbLedAnalyzerSettings : public AnalyzerSettings
 	AsyncRgbLedAnalyzerSettings();
```

Comment:
I'd like to add `noexcept` to all methods where possible. I'll check if we can support that. More details in the overall review notes.

Reply:
Okay, will try it.